code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -596,10 +615,8 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 
 	if(ruleset.pre_execute(roundstart_pop_ready))
 		threat_log += "[worldtime2text()]: Roundstart [ruleset.name] spent [ruleset.cost + added_threat]. [ruleset.scaling_cost ? "Scaled up [ruleset.scaled_times]/[scaled_times] times." : ""]"
-		if(ruleset.flags & ONLY_RULESET)
+		if(CHECK_BITFIELD(ruleset.flags, ONLY_RULESET))
 			only_ruleset_executed = TRUE
-		if(ruleset.flags & HIGH_IMPACT_RULESET)
-			high_impact_ruleset_executed = TRUE
```

Comment:
There needs to be some type of logic for roundstart rulesets to avoid injecting with other high impacts. Incursion is high impact but can co-occur with traitors/changeling since it doesn't have NO_OTHER_ROUNDSTART_RULESET.

If another high impact were added that also lacked NO_OTHER_ROUNDSTART_RULESET, it potentially occur alongside incursion

Reply:
This variable just got rolled over into the `high_impact_ruleset_active` proc, which is basically the same thing as this variable but takes dead-ness into account.